### ISSUE TITLE
BBOB data: strange costs

### ISSUE THREAD

**Issue by berndbischl** [berndbischl/coseal-algsel-benchmark-repo#63]
- Why do we have 0 costs for some steps?
- Are some of the costs really as big as 3686000 ?

**Comment by kerschke** [comment]
Yes. Some steps cost nothing because they only use the initial design (and those costs are already listed in the corresponding step).
And yes, some features were veeeery expensive - as I stated within my thesis. Maybe I should give you the thesis back again ;-)

**Comment by berndbischl** [comment]
> Yes. Some steps cost nothing because they only use the initial design 
> (and those costs are already listed in the corresponding step).

I got confused here. The FEATURES need the initial design step so there are no FEATURES without costs.

> And yes, some features were veeeery expensive - as I stated within my thesis.
> Maybe I should give you the thesis back again ;-)

Please give me a set of steps which are likely to work well for prediction but are somewhat cheap.
Like Marius did for SAT. So we can use them in the experiments.

**Comment by kerschke** [comment]
Actually, I am not sure how well the models would work without the two expensive feature steps (local search and curvature). However, as those two groups are soooo expensive, I suggest to drop them.

**Comment by berndbischl** [comment]
Please dont close when we dont have a final answer....

**Comment by berndbischl** [comment]
I still need to decide the step set

**Comment by kerschke** [comment]
Why? We talked about it last Friday... :-)

**Comment by kerschke** [comment]
@berndbischl Can this be closed now?

**Comment by berndbischl** [comment]
I will use this later to finally discuss bbob. whether it is completely ok.

**Comment by berndbischl** [comment]
We need to do 2 things:
- check data set
- decide default feature steps

**Comment by kerschke** [comment]
I thought, we've already decided about the default feature steps:
- initial_design
- meta
- convexity
- y_distr
- levelset

**Comment by berndbischl** [comment]
Well, maybe you proposed these, but - because I had no time - I never ran on BBOB again. 

It is also good that they are now written down here.

**Comment by kerschke** [comment]
They are also mentioned in the config-file :-)
